src/main/java/echochat/Deadline.java

```diff
@@ -0,0 +1,35 @@
+package echochat;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task{
+    protected String by;
+    protected LocalDateTime dt;
+
+    public Deadline(String by, String desc) {
+        super('D', desc);
+        this.by = by;
+        this.dt = parseDateTime(by);
+    }
+
+    /**
+     * Get description of Deadline.
+     * @return Task description with date included.
+     */
+    @Override
+    public String getDesc() {
```

Comment:
I'm not sure what Desc meant here, maybe consider changing to a more straightforward name?